string cup_type
string coffee_type
string milk_level
string sugar_level
string payment